public/v4/apps/activepieces.yml

```diff
@@ -0,0 +1,28 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        image: index.docker.io/activepieces/activepieces:$$cap_activepieces_version
+        environment:
+            AP_QUEUE_MODE: MEMORY
+            AP_DB_TYPE: SQLITE3
+            AP_FRONTEND_URL: http://$$cap_appname.$$cap_root_domain
+        volumes:
+            - $$cap_appname-data:/root/.activepieces
+        restart: always
+        caproverExtra:
+            containerHttpPort: '80'
+caproverOneClickApp:
+    variables:
+        - id: $$cap_activepieces_version
+          label: Activepieces Version
+          defaultValue: '0.12.2'
+          description: Check out their Docker page for the valid tags https://hub.docker.com/r/activepieces/activepieces/tags
+    instructions:
+        start: >-
+            This is a simple setup. For more advanced setup please check their repository and adjust app accordingly
+        end: >-
+            Please wait for 2 minutes. Activepieces is deployed and available as $$cap_appname, please change AP_FRONTEND_URL when you force https
+    displayName: Activepieces
+    isOfficial: true
+    description: Your friendliest open source all-in-one automation tool, designed to be extensible through a type-safe pieces framework written in Typescript.
```

Comment:
I recommend mentioning "Zapier Alternative" here. I bet lots of people want to see this app!

Reply:
will include the revision in my next contribution. Thanks!